evaluation/backtester.py

```
@@ -0,0 +1,710 @@
+        direction: str,
+        price: float,
+        timestamp: datetime
+    ):
+        """Close positions of a specific type."""
+        for position in list(self.positions):
+            if position.direction == direction:
+                self._close_position(position, price, timestamp)
+
+    def _update_positions(
+        self,
+        price: float,
```

Ruff (0.14.7)

Unused method argument: `price`

(ARG002)